R/contingencytables.R

```diff
@@ -202,39 +202,42 @@ ContingencyTables <- function(jaspResults, dataset, options, ...) {
   }
 }
 
-.crossTabLogOdds <- function(jaspResults, dataset, options, analyses, ready) {
-  if(!options$oddsRatio)
-    return()
 
-  for (i in 1:nrow(analyses)){
+.crossTabOdds <- function(jaspResults, dataset, options, analyses, ready){
+  if (!options$oddsRatioNormal)
+    return ()
+  
+  for (i in 1:nrow(analyses)) {
     analysis <- analyses[i,]
-    analysisContainer <- jaspResults[[paste0("container", i)]]
-    if (!is.null(analysisContainer[["crossTabLogOdds"]]))
+    analysisContainer <- jaspResults[[paste0("container",i)]]
+    if (!is.null(analysisContainer[["crossTabOdds"]]))
       next
-
+    
```

Comment:
Please avoid unnecessary whitespace. If you open the project file (`jaspFrequencies.Rproj`) then rstudio should by default remove those.